LiteFarmOrg/LiteFarm
# Lf 2892 user nominated data ie crops species and varietals

Jira Ticket: [LF-2892](https://lite-farm.atlassian.net/browse/LF-2892?atlOrigin=eyJpIjoiNWZkYThmNjQzY2M1NGE1MmI2ZTdlZTEyNjEzOWEwNjIiLCJwIjoiaiJ9) 

Description:
Adds nomination db architecture. Including:
- Database architecture creation
- Models for each table
- A controller for the overall nomination
- A route that includes create, update, delete methods
- A new farm access verification
- Updated server to include route

To test: 
Use Postman or similar api request option.
1. Create 
- POST to /nomination
- { auth: id_token, header: user_id, farm_id, body: user_id, farm_id, nomination_type="CROP_NOMINATION"}
- view in table if nomination and status log is present
2. Delete
- DELETE to /nomination/[nomination_id from previous step eg. 1]
- { auth: id_token, header: user_id, farm_id }
- check the nomination table and see that deleted=true in base object property

[LF-2892]: https://lite-farm.atlassian.net/browse/LF-2892?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
